#!/usr/bin/env python
PACKAGE = "project_1"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("wheelRadius", double_t, 0, "Wheels radius parameter", 0.069, 0.01, 1)
gen.add("halfLength", double_t, 1, "Wheel position along x (l) in meters", 0.195, 0.01, 1)
gen.add("halfWidth", double_t, 2, "Wheel position along y (w) in meters", 0.164, 0.01, 1)
gen.add("tickRes", int_t, 3, "Encoders resolution (N counts per revolution", 38, 1, 100)

exit(gen.generate(PACKAGE, "velocitiesTicks", "robotparameters"))